src/library/banshee/bansheefeature.cpp

```diff
@@ -61,6 +63,12 @@ QIcon BansheeFeature::getIcon() {
     return m_icon;
 }
 
+void BansheeFeature::bindSidebarWidget(WLibrarySidebar* pSidebarWidget) {
+    // store the sidebar widget pointer for later use in onRightClickChild
+    m_pSidebarWidget = pSidebarWidget;
+    qDebug() << "   BansheeFeature::bindSidebarWidget(" << pSidebarWidget << ")";
+}
+
```

Comment:
I'll move `bindSidebarWidget()` to baseexternallibrary and make it virtual there, so I can remove it from the other external features and have it only overwritten in iTunes as this seems to be the only external feature that has custom rightClick() function and only for the root item.